### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by FaNNboii** [sebastienvercammen/Pokemon-Go-Coords#7]
/var/www/html/pokesniper/discord-bot/node_modules/discord.js/lib/Structures/Server.js:5
var oldChannel=this.eventVoiceLeave(user);channel.members.add(user);user.voiceChannel = channel;if(oldChannel.id){this.client.emit("voiceLeave",oldChannel,user);this.client.emit("voiceSwitch",oldChannel,channel,user);}this.client.emit("voiceJoin",channel,user);};Server.prototype.eventVoiceStateUpdate = function eventVoiceStateUpdate(channel,user,data){if(!user.voiceChannel || user.voiceChannel.id !== channel.id){return this.eventVoiceJoin(user,channel);}if(!this.memberMap[user.id]){this.memberMap[user.id] = {};}var oldState={mute:this.memberMap[user.id].mute,selfMute:this.memberMap[user.id].self_mute,deaf:this.memberMap[user.id].deaf,selfDeaf:this.memberMap[user.id].self_deaf};this.memberMap[user.id].mute = data.mute;this.memberMap[user.id].selfMute = data.self_mute;this.memberMap[user.id].deaf = data.deaf;this.memberMap[user.id].selfDeaf = data.self_deaf;if(oldState.mute !== undefined && (oldState.mute != data.mute || o

TypeError: Cannot read property 'add' of undefined
    at Server.eventVoiceJoin (/var/www/html/pokesniper/discord-bot/node_modules/discord.js/lib/Structures/Server.js:5:58)
    at new Server (/var/www/html/pokesniper/discord-bot/node_modules/discord.js/lib/Structures/Server.js:4:5212)
    at /var/www/html/pokesniper/discord-bot/node_modules/discord.js/lib/Client/InternalClient.js:88:2098
    at Array.forEach (native)
    at WebSocket.websocket.onmessage (/var/www/html/pokesniper/discord-bot/node_modules/discord.js/lib/Client/InternalClient.js:88:2023)
    at WebSocket.onMessage (/var/www/html/pokesniper/discord-bot/node_modules/ws/lib/WebSocket.js:442:14)
    at emitTwo (events.js:87:13)
    at WebSocket.emit (events.js:172:7)
    at Receiver.onbinary (/var/www/html/pokesniper/discord-bot/node_modules/ws/lib/WebSocket.js:848:10)
    at /var/www/html/pokesniper/discord-bot/node_modules/ws/lib/Receiver.js:628:18

**Comment by sebastienvercammen** [comment]
@FaNNboii Please format your issue more appropriately. Tell me your environment, your settings, your current versions, a screenshot of your entire error (and/or a formatted log), and so on.

**Comment by KanekiiVII** [comment]
@sebastienvercammen , Is it possible to contact u via Skype?

**Comment by sebastienvercammen** [comment]
@KanekiiVII No, I don't give GitHub support via Skype.

You're welcome to just ask your question or if you get lucky you can find me on the PoGoDev Discord channel: https://discord.gg/qRZEb (link only available for 24 hours).

**Comment by KanekiiVII** [comment]
@sebastienvercammen K, thx for ur fast response. Is it possible to write or talk with u on this website? Because I got some questions, which I want to ask u.

**Comment by sebastienvercammen** [comment]
@KanekiiVII Yes, my Discord name is Seb. You can find me in the #general or #help channels.

**Comment by KanekiiVII** [comment]
@sebastienvercammen Ok, thanks for ur fast response bro! Keep it up ^^

**Comment by FaNNboii** [comment]
@sebastienvercammen what you mean with format more appropriately?
It runs over 24hrs fine on the Server. And today it comes up this message when i try to start it.
